app/sprinkles/admin/src/Controller/UserController.php

```diff
@@ -167,7 +167,7 @@ public function create(Request $request, Response $response, $args)
             $passwordRequest = $this->ci->repoPasswordReset->create($user, $config['password_reset.timeouts.create']);
 
             // If the password_mode is manual, do not send an email to set it. Else, send the email.
-            if (!isset($data['value'])) {
+            if ($data['password'] === '') {
```

Comment:
The test failure was actually due to the `isset` here - it's set to empty string if not set some lines earlier.